charts/alertmanager/values.schema.json

```diff
@@ -912,4 +953,3 @@
             }
         }
     }
-}
```

Comment:
this seems wrong

Reply:
@monotek Thanks for taking a look at this PR. My linter is saying that the JSON is valid, would you mind checking again?

Reply:
@monotek I have made changes to this file if you could re-review.